### PR TITLE
GRID-313: Allow a hidden default sort column

### PR DESCRIPTION
**Pull request by joneit**

### CODE REVIEW

**Comment by joneit** [comment]
This implements William's request to be able to set the (hidden) ID as the default sort column, to essentially group the data but without a visible group sorting.

**Comment by joneit** [comment]
Sorry this PR got cluttered up with by-id stuff. Unintended. :(

File: src/behaviors/Behavior.js [hunk 1]
```diff
@@ -1665,91 +1661,6 @@ var Behavior = Base.extend('Behavior', {
...
         return this.dataModel.getFilteredData();
     },
-
-    /**
-     *
-     * @param newDataRow
-     */
-    addDataRow: function(newDataRow) {
-        requeue.call(this, this.gridBehaviorChanged);
-        return this.dataModel.addRow(newDataRow);
```
Comment (Dwaynekj) [inline src/behaviors/Behavior.js]: This code is from the other PR right? You just continued off of that branch?

Comment (joneit) [inline src/behaviors/Behavior.js]: @Dwaynekj Actually no... There were no interdependencies so instead I separated the PRs so they both branched off of develop. But yes you are right, this deletion more properly belongs in the other PR. I will move it.

**Pull request merged**